docs/source_es/quicktour.mdx

```diff
@@ -62,11 +62,10 @@ En el siguiente ejemplo, usarás el [`pipeline`] para análisis de sentimiento.
 
 Instala las siguientes dependencias si aún no lo has hecho:
 
-```bash
-pip install torch
-===PT-TF-SPLIT===
-pip install tensorflow
-```
+<frameworkcontent>
+  <pt>```bash pip install torch ```</pt>
+  <tf>```bash pip install tensorflow ```</tf>
+</frameworkcontent>
```

Comment:
Let's kettp the multiline syntax here :-)
```suggestion
<frameworkcontent>
<pt>
```
bash
pip install torch
```
</pt>
<tf>
```bash
pip install tensorflow
```
</tf>
</frameworkcontent>
```
otherwise the commands won't be rendered in bash highlight